tests/callbacks/test_callbacks.py

```diff
@@ -1,24 +1,30 @@
 # Copyright 2022 MosaicML Composer authors
 # SPDX-License-Identifier: Apache-2.0
 
-import os
-from typing import Any, Callable, List
+from typing import Any, Callable, Dict
 
 import pytest
+from torch.utils.data import DataLoader
 
-import composer.callbacks
-import composer.loggers
-import composer.profiler
-from composer.callbacks.early_stopper import EarlyStopper
-from composer.callbacks.mlperf import MLPerfCallback
-from composer.callbacks.threshold_stopper import ThresholdStopper
+from composer.callbacks.callback_hparams import CallbackHparams, callback_registry
 from composer.core import Event
 from composer.core.callback import Callback
 from composer.core.engine import Engine
 from composer.core.state import State
-from composer.loggers import Logger, ObjectStoreLogger, WandBLogger
+from composer.loggers import Logger
```

Comment:
consolidate imports? with there was a package to do this automatically.. find the min hop import path..

Reply:
Interview question? :D

Done